lisa/features/serial_console.py

```diff
@@ -53,10 +60,27 @@ def get_console_log(
                 f.write(self._cached_console_log)
         return self._cached_console_log.decode("utf-8", errors="ignore")
 
-    def check_panic(self, saved_path: Optional[Path], force_run: bool = False) -> None:
+    def check_panic(
+        self, saved_path: Optional[Path], stage: str, force_run: bool = False
```

Comment:
Give stage an empty value "", so that it's optional, when there is only one time check panic in some cases.